fe/fe-core/src/main/java/com/starrocks/qe/DefaultCoordinator.java

```diff
@@ -530,6 +530,13 @@ private void prepareProfile() {
             if (!jobSpec.isEnablePipeline()) {
                 jobSpec.getQueryOptions().setEnable_profile(true);
             }
+            if (jobSpec.isBrokerLoad() && jobSpec.getQueryOptions().getBig_query_profile_threshold() == 0) {
```

Comment:
does `insert` need to enable profile automatically if `enable_profile` is false and running time is bigger than `default_big_load_profile_threshold_second`？

Reply:
no, the reason is insert stmt could be very complex, the resource usage(cpu/mem/network) of profile may out of control.
we need to distinguish the simple ingestion query from all insert stmt before we default enable big load profile.